### ISSUE TITLE
Add <endoftext> for every lines

### ISSUE THREAD

**Issue by PoodleWang** [salesforce/jaxformer#8]
https://github.com/salesforce/jaxformer/blob/9d6730e15b8cdccc77b7a898a27c37e8f77c14b3/preprocess/5_shuffe_concat_eos.py#L72

Hi, I am handling data.

**Comment by enijkamp** [comment]
Yes, this is the pre-processing code for ProGen2, which would require minor adjustment.

I will add an implementation and instructions for end-to-end fine-tuning soon. Unless you would like to send a PR.

Sorry for these delays.

**Comment by PoodleWang** [comment]
> @PoodleWang Did it got any process?

I solve this issue by myself.
